Project.toml

```
@@ -1,7 +1,7 @@
 name = "DelayDiffEq"
 uuid = "bcd4f6db-9728-5f36-b5f7-82caef46ccdb"
 authors = ["Chris Rackauckas <accounts@chrisrackauckas.com>"]
-version = "5.25.2"
+version = "5.25.3"
```

```suggestion
version = "5.26.0"
```

bounds change so minor.